src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -66,4 +71,14 @@ public void tankCane(double leftSpeed, double rightSpeed) {
     m_leftCaneMotor.set(ControlMode.PercentOutput, leftSpeed);
     m_rightCaneMotor.set(ControlMode.PercentOutput, rightSpeed);
   }
+
+  public void openClaw() {
```

Comment:
We may want to split this up into closeLeftClaw() / openLeftClaw() and closeRightClaw / openRightClaw()